README.markdown

```diff
@@ -99,7 +99,7 @@ To verify that nvm has been installed, do:
 command -v nvm
 ```
 
-which should output 'nvm' if the installation was successful. Please note that `which nvm` will not work, since `nvm` is a sourced shell function, not an executable binary.
+which should output 'nvm' if the installation was successful. Please note that `which nvm` will not work, since `nvm` is a sourced shell function, not an executable binary. Also note, you may need to close your existing terminal window and run the command in a new window.
```

Comment:
it was just something that had me confused for a little while so I thought I'd submit a PR. But fair enough, if I had just read the output, it would've been clear.